src/components/Select.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { SelectProps } from 'rendition';
+import styled, { withTheme } from 'styled-components';
+import asRendition from '../asRendition';
+import { radius } from '../theme';
+import { px, withProps } from '../utils';
+
+const Base = withProps<
+	{ emphasized?: boolean } & React.HTMLProps<HTMLSelectElement>
+>()(styled.select)`
+  border-radius: ${px(radius)};
+  height: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
+  font-size: inherit;
+  border: 1px solid ${props => props.theme.colors.gray.main};
+  padding-top: 0px;
+  padding-bottom: 0px;
+  padding-left: 16px;
+  padding-right: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
```

Comment:
Somehow the styles for `Base` ended up with two space indentation except from the lines that use `px` which are indented with two tabs !? :alien: :satellite: